Create junk.md (#5)
mathmanican/testing: junk.md

> adding a new file

Author: ```suggestion
adding a new file here.
```

> adding a new file here.

Author: ```suggestion
Adding a new file here.
```

want to test commentaty features

> just chekcing

Author: ```suggestion
just checking.
```

> just checking.

Author: ```suggestion
Just checking.
```

> Adding a new file here.
> want to test commentaty features
> Just checking.

Author: ```suggestion
Adding a new file here.
I want to test commentary features. 
Just checking.
How did it go?
```

> Adding a new file here.

Author: ```suggestion
Adding a new file.
```

I want to test commentary features. 

> Just checking.

Author: ```suggestion
Just checking how it works.
```

> How did it go?

Author: ```suggestion
How did it go with batches?
```
